lib/src/isolate_channel.dart

```diff
@@ -3,7 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:isolate';
+import 'isolate_stub.dart' if (dart.library.isolate) 'dart:isolate';
```

Comment:
We need to add the `// ignore: uri_does_not_exist` comment here or else it will fail on the web due to https://github.com/dart-lang/sdk/issues/34177

Reply:
Are we going to add associated tests to validate this works (doesn't crash)

Or do we already have those?

Reply:
I'm not sure what testing you're asking for - do you want me to add a test that validates it can be build with DDC?

Reply:
That's my concern. We don't have to, but my understanding is that we've had things blow up after publish here.

Reply:
Ha - good call @kevmoo

This import is not supported. https://github.com/dart-lang/build/blob/8df2826c81a417b44a05a2fc707c3a59481f3c27/build_modules/lib/src/module_library.dart#L107-L110

We either need to relax that restriction (which IIRC was somewhat arbitrary?) or pick a different approach here.